pom.xml

```diff
@@ -6,28 +6,31 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>2.26</version>
+		<version>4.13</version>
 	</parent>
 
 	<artifactId>conditional-buildstep</artifactId>
-	<version>1.3.7-SNAPSHOT</version>
+	<version>${revision}${changelist}</version>
 
 	<packaging>hpi</packaging>
 
 	<name>Conditional BuildStep</name>
 	<description>A buildstep wrapping any number of other buildsteps, controlling their execution based on a defined condition.</description>
 	<url>https://wiki.jenkins-ci.org/display/JENKINS/Conditional+BuildStep+Plugin</url>
 	<scm>
-		<connection>scm:git:git://github.com/jenkinsci/conditional-buildstep-plugin.git</connection>
-		<developerConnection>scm:git:ssh://git@github.com/jenkinsci/conditional-buildstep-plugin.git</developerConnection>
-		<url>https://github.com/jenkinsci/conditional-buildstep-plugin</url>
-	  <tag>HEAD</tag>
+		<connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+		<developerConnection>scm:git:ssh://git@github.com/${gitHubRepo}.git</developerConnection>
+		<url>https://github.com/jenkinsci/${gitHubRepo}</url>
+	  <tag>${scmTag}</tag>
   </scm>
 	<properties>
+		<revision>1.3.7</revision>
+		<changelist>-SNAPSHOT</changelist>
+		<gitHubRepo>jenkinsci/conditional-buildstep-plugin</gitHubRepo>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.javadoc.skip>true</maven.javadoc.skip>
-		<jenkins.version>1.580.1</jenkins.version>
-		<java.level>6</java.level>
+		<jenkins.version>2.176.4</jenkins.version>
```

Comment:
best to go to at least 2.204.6 if not later, that means there's no implied dependencies on detached plugins